components/script/devtools.rs

```diff
@@ -103,7 +103,7 @@ fn find_node_by_unique_id(
     documents.find_document(pipeline).and_then(|document| {
         document
             .upcast::<Node>()
-            .traverse_preorder()
+            .traverse_preorder(ShadowIncluding::No)
```

Comment:
This probably wants to be shadow_including, if it's for devtools, but can be a followup, with its own tests and such.